crates/topos/src/components/node/mod.rs

```diff
@@ -115,11 +115,21 @@ pub(crate) async fn handle_command(
             );
 
             // Load genesis pointed by the local config
-            let genesis = Genesis::new(
+            let genesis = match Genesis::new(
                 home.join("subnet")
                     .join(config.base.subnet_id.clone())
                     .join("genesis.json"),
-            );
+            ) {
+                Ok(genesis) => genesis,
+                Err(e) => {
+                    error!(
+                        "Failed to load subnet genesis file: {}\n Could not run node without \
```

Comment:
Can we change it to a more productive error, since it is read by a user? Like:

"Could not load `genesis.json` file on path XYZ. Please make sure to have a valid genesis.json file for your subnet in the `subnet/SUBNET_ID` folder."